src/Domains/Connectors/Google/Services/MapStaticApiService.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\Google\Services;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\ConnectException;
+use Illuminate\Support\Facades\Log;
+use Kanvas\Apps\Models\Apps;
+use Throwable;
+
+class MapStaticApiService
+{
+    public const DEFAULT_ZOOM = '20';
+    public const DEFAULT_SIZE = '600x400';
+    public const MAP_STATIC_API_URL = "https://maps.googleapis.com/maps/api/staticmap?";
+    /**
+     * Get the top 3 most relevant tags for a given message.
+     */
+    public static function getImageFromCoordinates(float $latitude, float $longitude, ?string $zoom = null): string
+    {
+        $app = app(Apps::class);
```

Comment:
@rwhite27 remember to pass the app as param to the service , don't relay on the global